packages/synapse-react-client/src/components/SetBasicAccessRequirementFields/AccessRequirementWikiInstructions.test.tsx

```diff
@@ -89,14 +89,17 @@ describe('AccessRequirementWikiInstructions', () => {
     await user.type(markdownField, newMarkdown)
     expect(markdownField).toHaveValue(newMarkdown)
 
+    // change getGetWikiPage since MarkdownSynapse will GET wiki rather than
+    // ...reading from react query cache
+    changeGetWikiPageHandlerOnce({ ...createdWikiPage, markdown: newMarkdown })
```

Comment:
Update the handler before clicking the save button, so there is not a timing issue related to when the handler is updated vs when the GET call is made